kernel/src/scan/state.rs

```diff
@@ -30,11 +29,19 @@ pub struct GlobalScanState {
 }
 
 /// this struct can be used by an engine to materialize a selection vector
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Default, Debug, Clone, PartialEq, Eq)]
 pub struct DvInfo {
     pub(crate) deletion_vector: Option<DeletionVectorDescriptor>,
 }
 
+impl From<DeletionVectorDescriptor> for DvInfo {
+    fn from(deletion_vector: DeletionVectorDescriptor) -> Self {
+        DvInfo {
+            deletion_vector: Some(deletion_vector),
+        }
```

Comment:
clippy wrangling nit:
```suggestion
        let deletion_vector = Some(deletion_vector);
        DvInfo { deletion_vector }
```